download.html

```diff
@@ -31,8 +31,11 @@ <h2 id="swanlake">{{ site.data.swanlake-latest.metadata.display-version }}</span
                <p>For more details, see the <a href="https://blog.ballerina.io/posts/2022-02-01-announcing-ballerina-2201.0.0-swan-lake/">Swan Lake GA release announcement </a> and <a href="/downloads/swan-lake-release-notes/">release note.</a></p><br/> 
 					<!-- <h2 id="swanlake">{{ site.data.swanlake-latest.metadata.display-version }} <span id="versionInfo">({{ site.data.swanlake-latest.metadata.release-date | date: "%B %e, %Y" }})</span></h2> -->
 					<div class="alert alert-info" style="border: 1px solid #20b6b0; background: #fff;">
-						<p>If you are already using Ballerina, use the <a href="/learn/tooling-guide/cli-tools/update-tool/">Ballerina Update Tool</a> to directly update to {{ site.data.swanlake-latest.metadata.display-version }} by running the command below. </p> <pre>bal dist pull 2201.0.0</pre>
-						<p>If your Update Tool is below version 0.8.14, use the <code class="highlighter-rouge language-plaintext">ballerina update</code> command.</p> 
+						<p>If you are already using Ballerina, you can use the <a href="/learn/tooling-guide/cli-tools/update-tool/">Ballerina Update Tool</a> to directly update to {{ site.data.swanlake-latest.metadata.display-version }}. To do this, first, run the command below to get the latest version of the Update Tool. </p> 
```

Comment:
@anupama-pathirage @sameerajayasoma Do we need to mention "you must execute" bal update since this has backward incompatible changes

Reply:
This is fine as we have introduced numbered steps